README.md

```diff
@@ -5,32 +5,33 @@
 Convert from Tensorflow to CoreML
 =================================
 
-Python package to convert from Tensorflow to CoreML format. To get the latest
-version of `tfcoreml`, please run:
+[coremltools](https://github.com/apple/coremltools) (Recommended approach)
+--------------------
+
+
+For converting TensorFlow models to CoreML format, the recommended approach is to use `coremltools 4.0` python package.
+
+Please see the  coremltools documentation on [Tensorflow conversion](https://coremltools.readme.io/docs/tensorflow-conversion) for example usage.
+
+To get the latest version of coremltools:
 
 ```shell
-pip install --upgrade tfcoreml
+pip install coremltools==4.0b1
```

Comment:
You don't need this, as this would go stale very quickly and you have already mentioned coremltools 4

Reply:
Removed.